src/olympia/reviews/tests/test_views.py

```diff
@@ -329,6 +330,88 @@ def test_add_logged(self):
         assert r.status_code == 200
         self.assertTemplateUsed(r, 'reviews/add.html')
 
+    def test_no_body(self):
+        for body in ('', ' \t \n '):
```

Comment:
nit, but a for loop with 2 items seems more readable as just the same code twice.